docs/user_guide/index.rst

```diff
@@ -9,4 +9,5 @@ User Guide
 
    fairness_in_machine_learning
    assessment
-   mitigation
\ No newline at end of file
+   mitigation
+   reductions
```

Comment:
reductions are mitigations, so this shouldn't be a separate section, right?

Reply:
I did start that way originally. However, I rapidly realised how much material it was going to add, and it seemed to make more sense to have it separate. Besides, this is more theoretical, whereas the mitigation section concentrates on how to use the classes.

Reply:
I fear this is complicated. As a user if I look at fairlearn.reductions and want a basic example showing me how to use that I can't really tell whether I want to look at mitigation or reductions. The way I've seen this so far was that this level of detail is too much for your average user, so it probably belongs elsewhere, such as the contributor guide as extra background. I definitely wouldn't put it into the tree as it is right now for the reasons above.